Add support for multiple Doxygen projects, add wifi bridge (#19341)
nrfconnect/sdk-nrf: doc/nrf/drivers/wifi/low_api_wifi.rst

Wi-Fi mode
##########

> The Wi-Fi mode is used for normal operation of the nRF70 Series device in, for example, STA mode or other modes that are supported in the future.

Reviewer: ```suggestion
The Wi-Fi mode is used for the normal operation of the nRF70 Series device in the :abbr:`STA (Station)` mode or other modes that will be supported in the future.
```

Reviewer: This suggestion seems to be missed.
